processing/src/main/java/io/druid/query/groupby/having/HavingSpec.java

```diff
@@ -32,7 +32,8 @@
     @JsonSubTypes.Type(name = "not", value = NotHavingSpec.class),
     @JsonSubTypes.Type(name = "greaterThan", value = GreaterThanHavingSpec.class),
     @JsonSubTypes.Type(name = "lessThan", value = LessThanHavingSpec.class),
-    @JsonSubTypes.Type(name = "equalTo", value = EqualToHavingSpec.class)
+    @JsonSubTypes.Type(name = "equalTo", value = EqualToHavingSpec.class),
+    @JsonSubTypes.Type(name = "dimSelector", value = DimensionSelectorHavingSpec.class)
```

Comment:
I will update doc. The code is change to address
https://github.com/druid-io/druid/issues/1984